conda/rapids-tpcx-bb.yml

```diff
@@ -8,20 +9,22 @@ dependencies:
   - python=3.7
   - cudatoolkit=10.2
   - cudf
+  - rmm
   - dask-cuda
   - dask-cudf
   - cuml
-  - scipy
   - ucx-py
   - ucx-proc=*=gpu
+  - blazingsql
+  - scipy
```

Comment:
Also looks extraneous?

Reply:
I'm not entirely sure, as we had it originally as well. @VibhuJawa @ayushdg any insight?

Reply:
Discussed offline. We need scipy for some cupy sparse functionality